src/main/java/seedu/medmoriser/model/qanda/QAndA.java

```diff
@@ -17,6 +17,8 @@
 
     // Identity fields
     private final Question question;
+    private boolean isQuiz = false;
+    private boolean isQuizAnswer = false;
```

Comment:
Not sure if these should be under QAndA? Also considering that the QuizCommand already has the isQuiz boolean, are we repeating the info here @joshruien?

Reply:
I was thinking about this and maybe instead modifying the QAndA itself can pass a copy of this QAndA to the component that's responsible for displaying the QAndA currently being quizzed?

Reply:
actually, i need a boolean field in qAndA to keep track what kind of cell to use for that particular qAndA.

i think it's possible to pass a copy of that QAndA, but then it means need to refactor the constructor to include isQuiz field, I find it a bit redundant haha (cuz everytime we initialize a qAndA need to indicate isQuiz)
Do you think we should refactor constructor?